src/pages/home/report/ReportActionsView.js

```diff
@@ -380,7 +379,7 @@ class ReportActionsView extends React.Component {
      * items have been rendered. If the number of actions has changed since it was last rendered, then
      * scroll the list to the end. As a report can contain non-message actions, we should confirm that list data exists.
      */
-    scrollToListBottom() {
+    scrollToBottomAndUpdateLastRead(){
```

Comment:
This is missing a space between the `()` and `{` our style guide is [here](https://github.com/Expensify/App/blob/main/STYLE.md#whitespace) if you need it.

Reply:
Done